waveorder/io/reader.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import os
+import zarr
+from tifffile import TiffFile
+
+
+class MicromanagerReader:
+
+    def __init__(self, folder):
+        """
+        loads ome-tiff files in folder into zarr or numpy arrays
+        Strategy:
+            1. read all files in folder
+            2. search for the file whose omexml does not contain the element tag "BinaryOnly"-> this is the master ome-tiff
+            3. each series represents a micro-manager stage position.  upon call for data, assign this data to a dict
+
+        :param folder: folder containing all ome-tiff files
+        :param reader: tifffile or aicsimageio
+        """
+
+        self._positions = {}
+        self.mm_meta = None
+        self.stage_positions = 0
+        self.height = 0
+        self.width = 0
+        self.frames = 0
+        self.slices = 0
+        self.channels = 0
+        self.chnames = []
+
+        self.master_ome_tiff = self._get_master_ome_tiff(folder)
+        print(self.master_ome_tiff)
+        self._set_mm_meta()
+
+    def _set_mm_meta(self):
+        with TiffFile(self.master_ome_tiff) as tif:
+            self.mm_meta = tif.micromanager_metadata
+
+            if self.mm_meta['Summary']['Positions'] > 1:
+                self.stage_positions = []
+                for p in range(self.mm_meta['Summary']['Positions']):
+                    print(f"appending {self.mm_meta['Summary']['StagePositions'][p]}")
```

Comment:
I recommend changing this line to 
```suggestion
                    print(f"appending {self.mm_meta['Summary']['StagePositions'][p]['Label']}")
```
This will give the position name instead of listing the long dictionary of elements which can clog up the print statement.

Reply:
It may be better to get rid of these appending statements altogether

Reply:
I agree with this suggestion. Printing detailed metadata upfront seems a bit confusing.